src/main/java/seedu/address/storage/Storage.java

```diff
@@ -29,4 +37,19 @@ public interface Storage extends AddressBookStorage, UserPrefsStorage {
     @Override
     void saveAddressBook(ReadOnlyAddressBook addressBook) throws IOException;
 
+    /**
+     * Exports the given {@link ObservableList} as a JSON file to the storage at filePathString.
+     * @param displayedList cannot be null.
+     * @param filePathString cannot be null.
+     * @throws IOException if there was any problem writing to the file.
+     */
+    static void exportDisplayedList(ObservableList<Person> displayedList, String filePathString) throws IOException {
```

Comment:
I am not sure about this, but I feel it is better to move this static method to StorageManager instead? Not sure, what do the others think

Reply:
Will do this once Deon's PR is merged